db/models/orderProduct.js

```diff
@@ -2,8 +2,22 @@
 
 // bcrypt docs: https://www.npmjs.com/package/bcrypt
 const bcrypt = require('bcryptjs')
-    , {STRING, VIRTUAL, BOOLEAN, INTEGER, DATE} = require('sequelize')
+    , {STRING, VIRTUAL, BOOLEAN, INTEGER, DATE, FLOAT} = require('sequelize')
 
 module.exports = db => db.define('orderProduct', {
-
+  quantity: {
+    type: INTEGER
+    // can we set default from the front end?
```

Comment:
front-end can set a default value in the form, but it won't go down to the database of course unless its submitted